manifests/s/schinagl/ln/2.9.3.4/schinagl.ln.locale.en-US.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Schinagl.ln
+PackageVersion: 2.9.3.4
+PackageLocale: en-US
+Publisher: schinagl.priv.at
+PackageName: ln advanced command line tool
+License: MIT License
+Copyright: Copyright © 1999-2021 Hermann Schinagl
+ShortDescription: 'is a swiss army knife for hardlinks, symbolic links, junctions when it comes to copying. Further more it provides a incremental copy mechanism called Delorean Copy '

```

```suggestion
ShortDescription: ln is a swiss army knife for hardlinks, symbolic links, junctions when it comes to copying. Further more it provides a incremental copy mechanism called Delorean Copy
```
I believe you wanted to write ln at the start? Also no need for single-quotes.